### PR TITLE
[feature] docker

### PR DESCRIPTION
**Pull request by ponsfrilus**
- Use node 20 "slim"
- install pnpm and pm2 using npm

### CODE REVIEW

**Comment by ponsfrilus** [comment]
This depends on #28

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# docker build -t epflsi/startpage .
+# docker run -it epflsi/startpage bash
+# docker run -e PORT=1337 -p 3000:1337 epflsi/startpage -d
+
+FROM node:20-slim
+
+# RUN apt-get -qq update && apt-get -qq -y --no-install-recommends install curl
+# RUN apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
+RUN npm install -g pnpm pm2
```
Comment (williambelle) [inline Dockerfile:17]: Do we want to version these packages?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Set internal port
+ENV PORT=1337
+# Open desired port
+EXPOSE 3000
+
+# It's a production Docker
+ENV NODE_ENV=production
+
+# Use pm2 to run the application
+ENTRYPOINT ["pm2-runtime", "./src/server.js"]
```
Comment (williambelle) [inline Dockerfile:38]: Maybe `USER node`?

File: Dockerfile (hunk 1 above)
Comment (ponsfrilus) [reply Dockerfile:17]: Maybe later, but right now I can't see why we would like to.

File: Dockerfile (hunk 2 above)
Comment (ponsfrilus) [reply Dockerfile:38]: Done

**Pull request merged**